src/modules/achievementAccess.ts

```diff
@@ -0,0 +1,17 @@
+import { UserRole } from 'prisma/prisma-client';
+
+import { AccessCheckResult, allowed, notAllowed } from '../utils/access';
+import { SessionUser } from '../utils/auth';
+
+import { tr } from './modules.i18n';
+
+export const achievementAccess = {
+    create: (sessionUser: SessionUser): AccessCheckResult => {
+        if (sessionUser.role === UserRole.ADMIN) return allowed;
+        return notAllowed(tr('Only admins can give achievement'));
```

Comment:
create

Creation should be in global access, since it doesn't need to check any existing data